README.md

```diff
@@ -8,4 +8,4 @@ The community is open to active open source maintainers on GitHub with a demonst
 
 _What is a maintainer?_ Maintainers are the stewards of an open source project. They spend much of their time thinking about the people and organizational aspects behind the project, beyond the project’s creative output (such as code). Maintainers might spend their time reviewing others’ contributions, responding to issues, evangelizing the project, or managing releases.
 
-We’re starting with a small, but growing, group of maintainers to ensure a good experience. If you’d like to join, [let us know](https://goo.gl/forms/0dgiaQXrmIV7OfsI3), and we’ll review it as we add more members.
+We’re starting with a small, but growing, group of maintainers to ensure a good experience. If you’d like to join, [let us know](maintainers.githubapp.com/apply), and we’ll review it as we add more members.
```

Comment:
I think you need to add the `https://` or it will be treated as a local link

Reply:
> I think you need to add the `https://` or it will be treated as a local link